### DISCUSSION TITLE
Fritz 7590 Mesh and no DNS resolution anymore :-(

### DISCUSSION THREAD

**Discussion by HeisemerBub** [AdguardTeam/AdGuardHome#3400] [Q&A]
Hello together,

I have a Fritzbox 7590 and have been working with AdGuard Home for a while. Before I switched the Fritzbox to Mesh (I have other Fritzboxes as Repaeaters at home), the names I had assigned in the Fritzbox were always displayed under "Clients"! 

After I switched to Mesh, AdGuard Home only shows me the IP in the "Clients" :-(

Is it still possible to set this somewhere despite the mesh function?

Thank you in advance for your brief feedback.

Greetings

**Comment by ameshkov** [comment]
> the names I had assigned in the Fritzbox were always displayed under "Clients"

What kind of names? Were these hostnames?

Maybe Fritzbox no more responds to PTR queries that we use to discover the clients' hostnames.

Probably, one of the ways to solve this would be to use AdGuard's DHCP server (you'll have to disable Fritzbox's DHCP).

**Comment by ainar-g** [comment]
Possibly related: #2961.
